modules/animation/src/animationmanager.cpp

```diff
@@ -213,18 +191,12 @@ std::unique_ptr<Interpolation> AnimationManager::getDefaultInterpolation(Propert
     if (interpolationIt != propertyToInterpolationMap_.end()) {
         interpolation = interpolationFactory_.create(interpolationIt->second);
         if (!interpolation) {
-            throw Exception("Default interpolation method for " + property->getClassIdentifier() +
-                                " was registered but the interpolation method was not added to the "
-                                "interpolation factory. @Developer: Please follow examples in "
-                                "animationmodule.cpp",
-                            IVW_CONTEXT);
+            LogError("Default interpolation method for " + property->getClassIdentifier() +
```

Comment:
should use << for the logging instead of +, or fmt::format

the LogError is redirecting the whole argument into a std::stringstream